src/components/reusable/page_header/styles.scss

```diff
@@ -0,0 +1,27 @@
+#page-header {
+  margin-bottom: 0.2rem;
```

Comment:
let's use 0.5rem, might be a bit much but we should be consistent, I've been trying to use either 1rem or 0.5rem in margin / padding across the board.

Reply:
No worries, can update it in the next PR 👍